.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2.1
+
+executors:
+  node:
+    docker:
+      - image: node:12
+        user: node
+  gcloud:
+    docker:
+      - image: google/cloud-sdk:alpine
+
+commands:
+  deploy-env:
+    description: "Deploy to a specific environment"
+    parameters:
+      sa_key_var:
+        type: string
+        default: "SA_KEY_JSON"
+      env:
+        type: string
+        default: "dev"
+      pr:
```

Comment:
Is this parameter likely to be used for this service? I suspect not.

Reply:
If we only have dev and prod then we will not need this. 
Although, not having any service running on the other environments doesn't seem right, as we would be making requests to a non-existant service